compatibility/common/src/main/java/org/projectnessie/tools/compatibility/internal/OlderNessieServersExtension.java

```diff
@@ -56,7 +56,7 @@ private void populateFields(ExtensionContext context, Object instance) {
 
     BooleanSupplier initializeRepository = () -> true;
     ServerKey serverKey =
-        ServerKey.forContext(context, version, "In-Memory", Collections.emptyMap());
+        ServerKey.forContext(context, version, "In-Memory", "Inmemory", Collections.emptyMap());
```

Comment:
I wouldn't mind if you change the name of the new in-memory to `In-Memory`, up to you.